src/Apps/Auction/Components/AuctionArtworkFilter.tsx

```diff
@@ -102,7 +106,7 @@ export const AuctionArtworkFilterRefetchContainer = createRefetchContainer(
     `,
   },
   graphql`
-    query AuctionArtworkFilterQuery(
+    query AuctionArtworkFilterRefetchQuery(
```

Comment:
I renamed this query to include 'refetch' (and distinguish from the query renderer one). But there were no other updates to this component (just _added_ the query renderer wrapper) - so I'm unsure why a spec on this refetch container is now failing in a weird way.